packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -1324,11 +1307,11 @@ export class ProjectLifecycleManager {
     if (!this._currentTestingType) {
       this.setCurrentTestingType('e2e')
       // TODO: Warn on this
-      // this.ctx.onWarning(this.ctx.error('TESTING_TYPE_NEEDED_FOR_RUN'))
+      // this.ctx.onWarning(getError('TESTING_TYPE_NEEDED_FOR_RUN'))
```

Comment:
Is this something we can do now?

Reply:
Good question, this is a run mode thing and I'm not sure what the behavior here should be.

I think @brian-mann said it should defaults to `e2e`, if none is passed probably for legacy reasons. 

I added this as a note awhile back to maybe revisit this, if we should always assume e2e if none is passed, or if we error if there are multiple setup and none are passed. If we did warn here it'd likely be a massive breaking change to all of our snapshots so I don't think we'd want to do in this PR... the diff here is only due to a find/replace of `this.ctx.error` -> `getError`

Reply:
Updated.